modules/specialized-guides/pages/large-deployments/hub-reqs.adoc

```diff
@@ -13,32 +13,112 @@ To set up a Hub installation, you require:
 
 == Hub Server
 
-Deploy the central {productname} Server, which acts as the Hub Server, with [command]``mgradm``.
+Deploy the central {productname} Server, which acts as the Hub Server, with the [command]``mgradm`` command.
+Add [option]``--hubxmlrpc-replicas 1`` to the [command]``mgradm install`` command line.
 For more information about deploying with [command]``mgradm``, see xref:installation-and-upgrade:container-deployment/suma/server-deployment-suma.adoc[].
 
 [NOTE]
 ====
 The Hub Server must not have clients registered to it.
-Clients should only be registered to the peripheral servers.
+Clients should only be registered to the Peripheral Servers.
 ====
 
 
 
+[[lsd-hub-reqs-certs]]
+=== SSL Certificates
+
+The Hub Server's and the Peripheral Servers' SSL certificates must be signed by the same CA.
+There are two ways to fulfill this requirement:
+
```

Comment:
This is the same requirement as here:
https://github.com/uyuni-project/uyuni-docs/blob/master/modules/specialized-guides/pages/large-deployments/hub-reporting.adoc#setup

Maybe it could be linked.

Reply:
thanks, I added a link.